File: udemy/pascal/projetos/Mediacondicional.pas
program Hello;
var media,A,B,C : real;
begin
write('informe a nota da primeira prova: ');
read(A);
write('informe a nota da segunda nota: ');
read(B);
write('informe a nota da terceira prova: ');
read(C);
media := (A+B+C)/3;
if media >= 6 then
    write('Passou com media:',media:6:2)
else
    write('nao passou, sua media foi:', media:6:2);
end.

File: udemy/java/condicionalMedia/src/condicionalMedia/Mediacondicional.java
package condicionalMedia;
import javax.swing.JOptionPane;
public class Mediacondicional {

	public static void main(String[] args) {
		float A,B,C,media;
		A = Float.parseFloat(JOptionPane.showInputDialog("informe a primeira nota"));
		B = Float.parseFloat(JOptionPane.showInputDialog("informe a segunda nota"));
		C = Float.parseFloat(JOptionPane.showInputDialog("informe a terceira nota"));
		media = (A+B+C)/3;
		if (media >= 6)
			JOptionPane.showMessageDialog(null,"Sua media foi:  "+media+"\nVoce passou!!");
		else 
			JOptionPane.showMessageDialog(null,"Sua media foi:  "+media+"\nVoce nao passou :(");
		

	}

}
